docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.9"
+
+services: 
+  allowedlist:
+    restart: always
+    image: dcspark/allowedlist-service:${ALLOWEDLIST_SERVICE_TAG:-0.0.7}
+    ports:
+      - ${PORT:-3000}:3000
+    environment:
+      - BRIDGE_CONTRACT_CHAIN_ID=${BRIDGE_CONTRACT_CHAIN_ID:-2001}
+      - CONTRACT_HOST=${CONTRACT_HOST:-https://rpc-mainnet-cardano-evm.c1.milkomeda.com}
+      - MAINNET=${MAINNET:-TRUE}
```

Comment:
do we want to hardcode any values in here? I think it should be always read from `env` (unless we want to have default deployment pointing to `mainnet`)

Reply:
Thanks for clarifying! Yeah, I just wanted to make sure we will overwrite those with `env` when `env` is specified :)